res/controllers/DummyDeviceDefaultScreen.qml

```diff
@@ -0,0 +1,306 @@
+import QtQuick 2.15
+import QtQuick.Window 2.3
+import QtQuick.Scene3D 2.14
+
+import QtQuick.Controls 2.15
+import QtQuick.Shapes 1.11
+import QtQuick.Layouts 1.3
+import QtQuick.Window 2.15
+
+import Qt5Compat.GraphicalEffects
+
+import Mixxx 1.0 as Mixxx
+import Mixxx.Controls 1.0 as MixxxControls
+
+import "." as Skin
+
+Item {
+    id: root
+
+    required property string screenId
+    property color fontColor: Qt.rgba(242/255,242/255,242/255, 1)
+    property color smallBoxBorder: Qt.rgba(44/255,44/255,44/255, 1)
+
+    property string group: "[Channel1]"
+    property var deckPlayer: Mixxx.PlayerManager.getPlayer(root.group)
+
+    function init(controlerName, isDebug) {
+        console.log(`Screen ${root.screenId} has started`)
+        switch (root.screenId) {
+            case "jog":
+                loader.sourceComponent = jog
+                break;
+            default:
+                loader.sourceComponent = main
+        }
+    }
+
+    function shutdown() {
+        console.log(`Screen ${root.screenId} is stopping`)
+        loader.sourceComponent = splash
+    }
+
+    // function transformFrame(input: ArrayBuffer, timestamp: date) {
+    function transformFrame(input, timestamp) {
+        return new ArrayBuffer(0);
+    }
+
+    Mixxx.ControlProxy {
+        group: root.group
+        key: "track_loaded"
+
+        onValueChanged: (value) => {
+            deckPlayer = Mixxx.PlayerManager.getPlayer(root.group)
+        }
+    }
+
+    Timer {
+        id: channelchange
+
+        interval: 2000
+        repeat: true
+        running: true
+
+        onTriggered: {
+            root.group = root.group === "[Channel1]" ? "[Channel2]" : "[Channel1]"
```

Comment:
do we care about 4-deck support here?

Reply:
The "Dummy Screen" mapping was only put there to allow anyone to test this feature without a S4 Mk3 device. I think this mapping can be removed before we merge

Reply:
that seems like it would be a useful feature, though?  would this help developers create screen designs for controllers of arbitrary sizes without needing to own the devices?  Also too, it might help with the creation of tests.

Reply:
(and we will need to test switchable 4 deck support, I would imagine)

Reply:
Perhaps this could go in the tools folder then? My concern being, we probably shouldn't ship this mapping as part of the the distribution. This could however be a good starting point for anyone wishing to test this feature and make the most of the screen rendering without using the physical device, which may help for design and testing. What do you think?  Maybe I can put that as a snippet in the `tools/README.md`?

Reply:
- [ ] Move `DummyMapping` to `tools` folder before merging